docs/addons/addon-types.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Types of addons'
+---
+
+Each Storybook addon is classified into two general categories, UI-based or Presets. Documented here are some of the features available for each type of addon. Use it as a reference when creating your next Storybook addon.
+
+## UI-based addons
+
+This particular type of addons allows you to customize Storybook's UI. If the addon you're developing falls into this category you can use some of the following elements to further customize the interface.
+
+### Panels
+
+This type of UI element allows you to add your own custom <code>Panel</code> to Storybook's UI. Currently the most common use case available throughout our ecosystem. Good examples that use this type of interface are: [@storybook/actions](../essentials/actions.md) and [@storybook/a11y](https://github.com/storybookjs/storybook/tree/next/addons/a11y).
+
+![Storybook panel](./storybook-panel.png)
+
+Below is the boilerplate code that you can use in your own addon to add a `Panel` in Storybook's UI.
```

Comment:
Rephrase this doc to describe what the reader should do in fewer words. Get straight to the point.

For example:
"Below is the boilerplate code that you can use in your own addon to add a `Panel` in Storybook's UI."

"Add a `Panel` to Storybook's UI using this boilerplate code."

Reply:
Fixed with the latest commit